editor.planx.uk/src/@planx/components/MapAndLabel/Editor.tsx

```diff
@@ -58,11 +77,50 @@ function MapAndLabelComponent(props: Props) {
                 name="fn"
                 placeholder={"Data Field"}
                 value={formik.values.fn}
+                onChange={formik.handleChange}
                 required
               />
             </InputRow>
           </InputGroup>
         </ModalSectionContent>
+        <ModalSectionContent title="Map Formatting">
```

Comment:
nit: I think most pages & headers throughout Planx favor only capitalising the letter of the first word, eg `Map formatting` (very big nit pick sorry)